php/php-rr-sdk.md

```diff
@@ -556,6 +538,25 @@ public function run(string $input): string
 ## Examples
 Following examples demonstrates features described above in a more realistic scenarios.
 
+### About Activities Stub
+Stable versions of SDK expected to have the default activity building stub to simplify developer life. While it is not
+totally clear how this stub will look like (until the API finalized) it can be constructed in workflow `__construct`
+method using (no context binding required).
+
+```php
+class SubscriptionWorkflow extends Workflow
+{
+    private ActivitiesInterface $activities;
+
+    public function __construct() 
+    {
+        $this->activities = new ActivityBuilder();    
```

Comment:
Consider using Java syntax:

`$this->activities = Workflow::newActivityStub(options);`